lib/zonefiles/2850

```diff
@@ -266,114 +275,79 @@ E 1 2917 1 3            CLOAK
 ? 0 5 0 E
 E 1 2923 35 15          GIRDLE
 ? 0 2 0 E
```

Comment:
This ? command should be removed, as the E command that was previously following it was converted to a Y command.
```suggestion
```